engine/src/main/java/io/camunda/zeebe/engine/processing/deployment/model/transformer/VariableMappingTransformer.java

```diff
@@ -150,7 +152,13 @@ private String asFeelContextExpression(
     return new MappingContextVisitor<>() {
       @Override
       public String onEntry(final String targetKey, final Expression sourceExpression) {
-        return targetKey + ":" + sourceExpression.getExpression();
+        String expression = sourceExpression.getExpression();
```

Comment:
🔧 local variable expression can be final by splitting the declaration and assignment by using an else statement in the if below to fit the assignment. We generally favor final, whenever possible.

Reply:
I updated my commit to use a final variable